palette.cabal

```
@@ -28,7 +28,7 @@ Library
                        Data.Colour.Palette.Types
                        Data.Colour.Palette.RandomColor
 
-  Build-depends:       base >= 4.2 && < 4.20,
+  Build-depends:       base >= 4.2 && < 4.21,
                        array >= 0.4 && < 0.6,
                        colour >= 2.3 && < 3.0,
                        containers >= 0.5 && < 0.7,
```

Does the `containers` bound need to be bumped?  I don't quite understand the [build failure with ghc 9.10](https://github.com/diagrams/palette/actions/runs/10653835942/job/29571614649?pr=9); in fact the following local build worked for me:
```
cabal build --with-ghc ghc-9.10.1
```


```suggestion
                       containers >= 0.5 && < 0.8,
```

---

The currently released version of `palette` on Hackage in fact has `containers < 0.8`.